README.md

```diff
@@ -2,31 +2,110 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Define Coupling and Cohesion
+- Explain tight coupling and loose coupling
+- Explain high cohesion and low cohesion
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+When we create an application in Java, we want to make sure that it is easy
+to maintain. Far too often do programmers spend more hours needed updating code.
```

Comment:
Consider rewording second sentence of this paragraph.
Maybe something like this?

> Far too often do programmers spend more hours updating code than needed.

Reply:
ok fixed.